public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -294,6 +295,9 @@ export function GrafanaEvaluationBehaviorStep({
         {/* Show the pending period input only for Grafana alerting rules */}
         {isGrafanaAlertingRule && <ForInput evaluateEvery={evaluateEvery} />}
 
+        {/*Show the keepFiringFor input only for Grafana alerting rules*/}
+        {isGrafanaAlertingRule && <KeepFiringFor evaluateEvery={evaluateEvery} />}
```

Comment:
nit: perhaps unrelated to this PR; but does this also exist in Mimir?

Reply:
yes, but we cannot use it in the form , 👀 [comment](https://github.com/grafana/grafana/pull/75163#issuecomment-1730129563)